src/nvim/terminal.c

```diff
@@ -278,15 +293,16 @@ void terminal_close(Terminal *term, int status)
   term->closed = true;
 
   if (status == -1 || exiting) {
-    // If status is -1, this was called by close_buffer(buffer.c).  Or if
-    // exiting, we must inform the buffer the terminal no longer exists so that
-    // close_buffer() doesn't call this again.
+    // If status is -1 so this was called by close_buffer(), or if inside
+    // process_teardown(), we must inform the buffer the terminal no longer
+    // exists so that close_buffer() doesn't call this again.
```

Comment:
```suggestion
    // If status is -1, this was called by close_buffer() or inside
    // process_teardown().  We must inform the buffer the terminal no longer
    // exists so that close_buffer() doesn't call this again.
```

Reply:
Well, this is not what I wanted to mean.